src/hotspot/share/runtime/threads.cpp

```diff
@@ -425,6 +425,19 @@ void Threads::initialize_jsr292_core_classes(TRAPS) {
   }
 }
 
+// One-shot PeriodicTask subclass for reading release file
+class ReadReleaseFileTask : public PeriodicTask {
+ public:
+  ReadReleaseFileTask(size_t interval_time) : PeriodicTask(interval_time) {}
```

Comment:
```suggestion
  ReadReleaseFileTask() : PeriodicTask(100) {}
```
The "delay" can just be hard-wired here.